content/articles/top-container-orchestration-tools/index.md

```diff
@@ -0,0 +1,137 @@
+
+If you have applications to develop, deploy and run, containers come in handy. With containers, applications are moved seamlessly between servers. Therefore, you need a management system that automatically runs containers, shuts them down when needed, and controls how containers access resources. That is where orchestration tools come in.
```

Comment:
Add 1 - 2 lines about what containers are, and link external articles if possible.